runelite-client/src/main/java/net/runelite/client/plugins/inventorytagger/ItemOutline.java

```diff
@@ -41,26 +58,28 @@ public BufferedImage getPicture()
 		SpritePixels itemSprite = client.createItemSprite(id, 1, this.stroke, 0, 0, true, 710);
 		itemSprite.setOutline(16777215);
 
-		Graphics2D graphics2D = image.createGraphics();
+		BufferedImage generatedPicture = new BufferedImage(36, 36, BufferedImage.TYPE_INT_ARGB);
```

Comment:
Oh, I thought this was constant. nevermind, your previous declaration of this field was correct